src/Tizen.Uix.VoiceControlManager/Tizen.Uix.VoiceControlManager/ErrorOccurerdEventArgs.cs

```diff
@@ -0,0 +1,169 @@
+
+                case ErrorCode.TimedOut:
+                    {
+                        Why = VoiceError.TimedOut;
+                        break;
+                    }
+
+                case ErrorCode.RecorderBusy:
+                    {
+                        Why = VoiceError.RecorderBusy;
+                        break;
+                    }
+
+                case ErrorCode.PermissionDenied:
+                    {
+                        Why = VoiceError.PermissionDenied;
+                        break;
+                    }
+
+                case ErrorCode.NotSupported:
+                    {
+                        Why = VoiceError.NotSupported;
+                        break;
+                    }
+
+                case ErrorCode.InvalidState:
+                    {
+                        Why = VoiceError.InvalidState;
+                        break;
+                    }
+
+                case ErrorCode.InvalidLanguage:
+                    {
+                        Why = VoiceError.InvalidLanguage;
+                        break;
+                    }
+
+                case ErrorCode.EngineNotFound:
+                    {
+                        Why = VoiceError.EngineNotFound;
+                        break;
+                    }
+
+                case ErrorCode.OperationFailed:
+                    {
+                        Why = VoiceError.OperationFailed;
+                        break;
+                    }
+
+                case ErrorCode.OperationRejected:
+                    {
+                        Why = VoiceError.OperationRejected;
+                        break;
+                    }
+
+                case ErrorCode.IterationEnd:
+                    {
+                        Why = VoiceError.IterationEnd;
+                        break;
+                    }
+
+                case ErrorCode.Empty:
+                    {
+                        Why = VoiceError.Empty;
+                        break;
+                    }
+
+                case ErrorCode.InProgressToReady:
+                    {
+                        Why = VoiceError.InProgressToReady;
+                        break;
+                    }
+
+                case ErrorCode.InProgressToRecording:
+                    {
+                        Why = VoiceError.InProgressToRecording;
+                        break;
+                    }
+
+                case ErrorCode.InProgressToProcessing:
+                    {
+                        Why = VoiceError.InProgressToProcessing;
+                        break;
+                    }
+
+                case ErrorCode.ServiceReset:
+                    {
+                        Why = VoiceError.ServiceReset;
+                        break;
+                    }
+            }
+            Message = msg;
+        }
+
+        /// <summary>
+        /// The error value.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public VoiceError Why
```

Comment:
I recommend to use Exception instead error code,

```suggestion
        public Exception Why
```

Reply:
Fixed.